### PR TITLE
Solve the issue 476

### PR DESCRIPTION
**Pull request by gdaguin**
- Reduce the size of the images as much as possible : couldn't do it much (for the non flat ones) because the images are also used in the logged in home screens : reduce the app size of 10 Mb
- Renaming files from v0x-waves-1100 to v0x-waves => No need to talk about the height of the images
- bk-flat files size drastic update => no need to have large files because that's row files with the same color every time
- code simplification to know if the theme is flat or not

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update
- Refactorization

## Material used:

Please delete options that are not relevant.
- iOS (Smartphone/Tablet)
- Android (Smartphone/Tablet)
- Browser
- Linux
- Windows
- iOS (Mac)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Test A
- Test B

### Patrol

Please list here the tests created in Patrol for this pull request.

## Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/ui/widgets/dialogs/theme_dialog.dart [hunk 1]
```diff
@@ -70,16 +67,8 @@ extension ThemePickerItemExt on PickerItem {
...
-        themeOption == ThemeOptions.darkFlat ||
-        themeOption == ThemeOptions.emeraldGreenFlat ||
-        themeOption == ThemeOptions.honeyOrangeFlat ||
-        themeOption == ThemeOptions.navyBlueFlat ||
-        themeOption == ThemeOptions.pearlGreyFlat ||
-        themeOption == ThemeOptions.fireRedFlat ||
-        themeOption == ThemeOptions.sapphireBlueFlat ||
-        themeOption == ThemeOptions.seaGreenFlat) {
+    // Flat theme
+    if (themeOption.toString().toLowerCase().endsWith('flat')) {
```
Comment (Chralu) [inline lib/ui/widgets/dialogs/theme_dialog.dart]: Simplifying this code is a pretty good idea 👍

I'm not an absolute fan of relying on the `ThemeOptions` values name. Indeed, those values are not strictly formatted.
What about adding a `ThemeOptions::isFlat` property (using [Dart enhanced enums](https://dart.dev/language/enums#declaring-enhanced-enums)) ?

@gdaguin This is a discussion-opened suggestion.

Comment (gdaguin) [reply lib/ui/widgets/dialogs/theme_dialog.dart]: That is way better, I didn't know this property. Thank you !

**Comment by redDwarf03** [comment]
Hello Guillaume. Thank you for your first contribution on the Archethic Wallet project !

**Comment by redDwarf03** [comment]
It might be interesting to break down the images by density
See: https://docs.flutter.dev/ui/assets-and-images#resolution-aware

Let's see if we can use this PR to deal with the subject or do a dedicated PR at the next issue 
https://github.com/archethic-foundation/archethic-wallet/issues/491

The advantage of having a single PR for the 2 issues is to have a single acceptance test session.

@gdaguin @Chralu for opinion

**Comment by gdaguin** [comment]
Do you have the original svg files ? It might be a better solution to apply filter to images - depending on the theme color, instead of duplicate all the images every time. We can try that to decrease the app overall size I think.

**Comment by redDwarf03** [comment]
> Do you have the original svg files ? It might be a better solution to apply filter to images - depending on the theme color, instead of duplicate all the images every time. We can try that to decrease the app overall size I think.

Sadly, it was not svg but png files...

**Comment by gdaguin** [comment]
So I think the best idea is to split the two PRs into two dedicated reviews because that's not the point of this one. It can be complex to know where we are if we put everything in one, I guess.

**Review by redDwarf03 — APPROVED** [review]

**Pull request merged**